core/src/main/java/c/org/rajawali3d/control/gles/GlesRenderControl.java

```diff
@@ -0,0 +1,10 @@
+package c.org.rajawali3d.control.gles;
+
+import c.org.rajawali3d.control.RenderControl;
+
+/**
+ * @author Randy Picolet
+ */
+
+public interface GlesRenderControl extends RenderControl {
```

Comment:
I know that this additional interface level is specifically to handle Vulkan vs GLES, but you should document that so it is clear.

Reply:
Forgot this existed. I'm not actually sure its needed, and I'm really sure it should not extend `RenderControl`...

Reply:
If it doesn't serve a purpose I'm fine with nuking it. One less interface to track

Reply:
Gone